bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue

```diff
@@ -443,12 +445,12 @@ export default {
         }).open()
       })
     },
-    doZWaveAction (action) {
+    doConfigAction (action) {
       let thing = this.thing
       let save = this.save
       if (action.type !== 'BOOLEAN') return
       this.$f7.dialog.confirm(
-        `${action.label}?`,
+        `${action.description ?? action.label}?${action.verify ? '<p><small><strong class="text-color-yellow">WARNING:</strong>&nbsp;This action may be dangerous!</small></p>' : ''}`,
```

Comment:
```suggestion
        `${action.description ?? action.label} ? ${action.verify ? '<p><small><strong class="text-color-yellow">WARNING:</strong>&nbsp;This action may be dangerous!</small></p>' : ''}`,
```
I'm not sure I understand this...

Reply:
Oh. and the abovementioned code was not a ternary IF statement! It is just a good old questionmark (to be rendered as-is in the output :) ).

Reply:
> The "actions" behaviour of Main UI does ask for a confirmation for all the actions, of course... so I figured to bring back at least some of the "verify" semantics, and add a notion of "hey, the binding developer considered this dangerous".

Wow I completely forgot this "verify" attribute existed... 🤯

Fully agree with your assessment - this particular line of code I considered quite "convoluted" though with the muliple ternary expressions. If I understand it right it does display the description or label, or when there's none, displays a warning message *if* the verify flag is true? Is that your intent?

Reply:
> or when there's none....

As per my 2nd comment... the `?` here is not a ternary expresssion, actually, rather the same questionmark that existed in the original. The intent is to:
  1.  Display description (and fallback to label if none)
  1. (orthogonal to the above): If `verify==true` --> add an extra warning.

I now see how poorly it reads code-wise (it was much better in IDE, with coloring, etc.). Unfortunately, I have literally never written anything in vue, so while I assume using some templated text/conditional controls would make it more readable, it would take me some extra learning to find the syntax out. Do you happen to have any tips in particular?

Reply:
Extracted to separate variables - hope it'll be more readable now.